docs/configuration/schema.yaml

```diff
@@ -5,176 +5,184 @@ properties:
     description: |-
       Stores your Redocly Workflows organization ID that the `push` command uses to push API definition files to the API registry.
       If an organization ID is not passed explicitly in command-line arguments, `push` will look for it in `redocly.yaml`.
-
-      To find the organization ID:
-
-        1. Log into Workflows.
-        2. Access the **API registry** page.
-        3. In your browser's address bar, find the page URL.
-        4. The segment after `app.redocly.com/org/` is the organization ID.
-
-      For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+      Find out more about [the `organization` section](https://redocly.com/docs/cli/configuration/#organization).
   apis:
+    type: object
+    description: |-
+      Lets you configure one or more API definition files.
+      This gives you the flexibility to reference specific files in commands, and configure each file at a granular level.
+      We have a lot more information about [the `apis` section](https://redocly.com/docs/cli/configuration/#apis), plus lots of examples.
     additionalProperties:
       x-additionalPropertiesName: "{name}@{version}"
       type: object
       minItems: 1
       required:
         - root
       description: |-
-        Specifies the name and version of an API associated with the root API definition with the pattern `{name}@{version}`.
-        If the version is omitted, Redocly apps interpret it as 'latest' by default.
+        Specifies the name and version of an API definition file that is associated with the root API file using the pattern `{name}@{version}`.
+        If `@version` is not provided, Redocly apps interpret a file as being `latest`.
       properties:
         root:
           type: string
-          description: "Path to the root API definition file of the specified API."
+          description: "The path to the root API definition file of the specified API."
         labels:
           type: array
-          description: "Optional labels assigned to the specified API in the Redocly API registry. The labels must exist in the Redocly Workflows organization in order to apply them."
+          description: "Assigns Redocly Workflows organization-wide labels to a definition file."
           items:
             type: string
         lint:
           type: object
           description: |-
-            Configures linting options for the specified API.
-            Supports the same format and options as the global lint section described further in the schema.
+            Defines linting rules at the file level. Supports the same format and options as the global `lint` section.
```

Comment:
It's actually for the specified API. Two APIs can point to the same file and have different linting options configured.